wiki/Ranking_criteria/osu!catch/en.md

```diff
@@ -65,11 +65,9 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 
 | Difficulty | Basic-snapped dash | Higher-snapped dash | Basic-snapped hyperdash | Higher-snapped hyperdash |
 | :-- | :-- | :-- | :-- | :-- |
-| **Cup** | - | - | - | - |
-| **Salad** | 250 ms or higher | 125-249 ms | - | - |
-| **Platter** | 125 ms or higher | 62-124 ms | 250 ms or higher | 125-249 ms |
-| **Rain** | 125 ms or higher | 62-124 ms | 125 ms or higher | 62-124 ms |
-| **Overdose** | - | - | - | - |
+| **Salad** | 250 ms or slower | 125–249 ms |  |  |
```

Comment:
This shouldn't be changed, since we're talking about time, not speed. Keep the osu!catch terminology that uses absolute time (ms) as-is.